src/docfx/lib/HtmlUtility.cs

```diff
@@ -18,6 +18,54 @@ internal static class HtmlUtility
     {
         public delegate void TransformHtmlDelegate(ref HtmlReader reader, ref HtmlWriter writer, ref HtmlToken token);
 
+        public static readonly string[] HtmlMetaHidden = new[]
+        {
+            "titleSuffix",
+            "contributors_to_exclude",
+            "helpviewer_keywords",
+            "dev_langs",
+            "f1_keywords",
+            "api_scan",
+            "layout",
+            "open_to_public_contributors",
+            "title",
+            "absolutePath",
+            "original_content_git_url_template",
+            "fileRelativePath",
+            "internal_document_id",
+            "product_family",
+            "product_version",
+            "redirect_url",
+            "redirect_document_id",
+            "toc_asset_id",
+            "content_git_url",
+            "area",
+            "theme",
+            "theme_branch",
+            "theme_url",
+            "is_active",
+            "publish_version",
+            "canonical_url",
+            "is_dynamic_rendering",
+            "need_preview_pull_request",
+            "moniker_type",
+            "is_significant_update",
+            "serviceData",
+            "github_contributors",
+            "is_hidden",
+        };
+
+        public static readonly Dictionary<string, string> HtmlMetaNames = new()
```

Comment:
nit: IReadOnlyDictionary

Reply:
Want to be compatible  with https://github.com/dotnet/docfx/blob/2146708759fad1b02b5f6e2bac48e202efecb8e6/src/docfx/lib/HtmlUtility.cs#L39